src/odemis/acq/acqmng.py

```diff
@@ -25,7 +25,7 @@
 
 
 from __future__ import division
-import numpy
+from numpy import flipud
```

Comment:
I'd rather have just `numpy` imported, and write explicitly `numpy.flipud()`, otherwise the function seems a little coming out of a magical place.